src/jenkins/BuildManifest.groovy

```diff
@@ -27,10 +27,24 @@ class BuildManifest implements Serializable {
         }
     }
 
+    class Components implements Serializable {
+        String dist
+
+        Components(Map data) {
+            this.dist = data.artifacts.dist
+        }
+
+        String getDistPackageLocation() {
+            return this.dist[0]
```

Comment:
Let’s be a little defensive and raise an error if dist has 0 or more than 1 item?